assessment.md

```diff
@@ -13,81 +13,512 @@
+  documentation, definition, and maintenance in the given design process
 * Software architecture
+  * N = There is no concrete structure and the master branch is broken
+  * I = There are small amounts of structure to the code where it may or may not
+  run correctly, and the master branch is broken
+  * A = There is a blueprint for the software system that lacks completion, and the
+  master branch is not broken
+  * G = There is a completed blueprint for the software system and a working master
+  branch
+  * E = The software system contains high-level structures that contain software
+  elements in a complex and complete blueprint of the software system, and the
+  master branch has no errors and contains completed code
 * Software design
+  * N = No problem solving steps taken
+  * I = Initial steps are taken to conceptualize, frame, and implement systems
+  * A = Some software design is visible to the point where a software solution looks
+  to be developed soon
+  * G = A software solution is created that includes both an algorithm design and
+  a high-level architecture design
+  * E = All goals and expectations of the design are met and compatibility, reliability,
+  reusability, and scalability are included    while a software solution is ultimately
+  created that includes both an algorithm design and a high-level architecture design
 * Software documentation
+  * N = No explanation or comments about how to use or how the software works
+  * I = A few comments in the code that are not detailed
+  * A = Documentation has a bit of detail for every main part of the program
+  * G = Documentation is detailed and visible for most of the software within the
+  source code
+  * E = Explains in great detail how to use and operate the computer software
+  within the source code
 * Programming styles
+  * N = Programming style is not set and always changing around during the program
+  * I = Not a clear programming style but has some implementation of general styles
+  * A = Some clarity of a programming style with multiple styles implemented
+  * G = Programming style is implemented well with a few variances throughout
+  * E = One distinct style of programming that is clear and easy to understand
 * Managing software complexity
+  * N = The code does not run properly, files are not in the right place, or some
+  other part of the project has been completely broken
+  * I = Little effort is put forth to keep code clean or organized, some code may
+  not work properly
+  * A = Basic organization is achieved, no major parts of the projects are
+  affected by any lax in management
+  * G = The project is logically and effectively organized with no issues in the
+  code resulting from poor organization
+  * E = Everything is exceedingly well organized, with no excess or lack of files,
+  folders, branches, forks, etc.
 * Mitigating software risk
+  * N = No procedure is followed, the master branch is broken, or files are corrupted
```

Comment:
I'm not sure this section correctly identifies good mitigation techniques, only the outcomes. This may work, but identifying techniques is probably fairer. A good, short guide on the subject I found is [here](https://www.test-institute.org/What_Is_Software_Risk_And_Software_Risk_Management.php). Of course, @gkapfham, @toccinAC, and @sutterj, feel free to correct me.

Reply:
Pull Request https://github.com/huangs1/assessment-professional-skills/pull/20 should have fixed this issue